src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import AppRoutes from "./routes";
 import "./assets/css/App.scss";
 import "./assets/authAssets/socialAuth.css";
+import "./assets/articleAssets/rating.scss";
```

Comment:
import both the scss files in the component that will be using it not in App.js,